### ISSUE TITLE
Tensorflow

### ISSUE THREAD

**Issue by Skyle81** [moeinp70/stock#2]
I know I am late, but I have errors with these imports: 
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import LSTM, Dense, Dropout
No matter what I do it keeps saying that tensorflow cannot be resolved. I tried changing my interpreter, created a .venv, recreated a new .venv but nothing seems to work. Every requirements are installed and up to date. Do you have any idea how to fix that?
